app/templates/head.hbs

```diff
@@ -0,0 +1,18 @@
+<link rel="apple-touch-icon" sizes="57x57" href="{{rootURL}}assets/favicon/apple-icon-57x57.png">
+<link rel="apple-touch-icon" sizes="60x60" href="{{rootURL}}assets/favicon/apple-icon-60x60.png">
+<link rel="apple-touch-icon" sizes="72x72" href="{{rootURL}}assets/favicon/apple-icon-72x72.png">
+<link rel="apple-touch-icon" sizes="76x76" href="{{rootURL}}assets/favicon/apple-icon-76x76.png">
+<link rel="apple-touch-icon" sizes="114x114" href="{{rootURL}}assets/favicon/apple-icon-114x114.png">
+<link rel="apple-touch-icon" sizes="120x120" href="{{rootURL}}assets/favicon/apple-icon-120x120.png">
+<link rel="apple-touch-icon" sizes="144x144" href="{{rootURL}}assets/favicon/apple-icon-144x144.png">
+<link rel="apple-touch-icon" sizes="152x152" href="{{rootURL}}assets/favicon/apple-icon-152x152.png">
+<link rel="apple-touch-icon" sizes="180x180" href="{{rootURL}}assets/favicon/apple-icon-180x180.png">
+<link rel="icon" type="image/png" sizes="192x192"  href="{{rootURL}}assets/favicon/android-icon-192x192.png">
+<link rel="icon" type="image/png" sizes="32x32" href="{{rootURL}}assets/favicon/favicon-32x32.png">
+<link rel="icon" type="image/png" sizes="96x96" href="{{rootURL}}assets/favicon/favicon-96x96.png">
+<link rel="icon" type="image/png" sizes="16x16" href="{{rootURL}}assets/favicon/favicon-16x16.png">
+<link rel="manifest" href="{{rootURL}}assets/favicon/manifest.json">
```

Comment:
You are right, I just created a new ticket #26 to start using this awesome addon.

Thanks for your tip